pattern-library/sass/patterns/_tables.scss

```diff
@@ -23,3 +24,148 @@ $table-border-color:                        palette(grayscale, x-light) !default
 // ----------------------------
 // #GLOBAL
 // ----------------------------
+
+
+%highlight {
+  background-color: palette(warning, x-trans);
+}
+
+%edx-table-base {
```

Comment:
Why the ``edx-`` namespacing here? We don't do it elsewhere and I understand tables being pretty common, but is this the place to break convention. If so, I can retro-fit it into other patterns. Other places seem to namespace pretty heavily (see http://www.lightningdesignsystem.com/components/buttons), so its not a crazy idea, just more weight on remembering the prefix and small amounts of file size when done across places.